eos/rare-b-decays/b-to-kstar-gamma-bfs2004_TEST.cc

```diff
@@ -156,6 +184,14 @@ class BToKstarGammaBobethCompatibilityTest :
             };
 
             Parameters p = Parameters::Defaults();
+            //p["CKM::abs(V_ub)"] =  0.003631275231633653;
```

Comment:
I had a look on this. If you use default CKM, then the test seems to fail with the first observable, which is BR. So this is what one would expect.

If you uncomment the values of the CKM elements, then the test for BR runs through, but the test for the next observable fails. This observable is S. 

Can you confirm this?

Reply:
I concur... I am investigating